server/routes/v1.routes.js

```diff
@@ -1,2 +1,16 @@
 const express = require('express');
 const Router = express.Router();
+
+const logController = require('../controllers/logController');
+
+Router.post('/log/store', logController.storeLog, (req, res, next) => {
+  const { logStored } = res.locals;
+
+  return res.status(201).json({
+    status: true,
+    log: logStored,
+    message: 'storeLog middleware: Successfully stored log into DB',
```

Comment:
Can you remove the storeLog middleware from here. The client doesn't want to know that detail implementation.